test/common/index.js

```diff
@@ -91,12 +91,25 @@ async function checkBuildType (buildType) {
 }
 
 async function whichBuildType () {
-  let buildType = 'Release';
-  if (await checkBuildType(buildTypes.Release)) {
-    buildType = buildTypes.Release;
-  }
-  if (await checkBuildType(buildTypes.Debug)) {
-    buildType = buildTypes.Debug;
+  let buildType;
+  const envBuildType = process.env.BUILD_TYPE;
```

Comment:
```suggestion
  const envBuildType = process.env.BUILD_TYPE || 'Release';
```

Reply:
Suggestion in meeting from @vmoroz how about `process.env.NODE_API_BUILD_CONFIG`